### PR TITLE
hub command

### PR DESCRIPTION
**Pull request by brpaz**
hub is a command-line wrapper for git that provides some nice extra commands for interacting with Github. see https://hub.github.com/ for more details.

In this PR I have added the basic hub commands.

### CODE REVIEW

File: pages/common/hub.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Hub
```
Comment (igorshubovych) [inline pages/common/hub.md]: should be lowercase, same as command name

**Comment by igorshubovych** [comment]
I would probably mention that the same commands are available under `git` too.

**Comment by brpaz** [comment]
OK. I have changed to command to lowercase and also add a note about using "git" instead of "hub"

File: pages/common/hub.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# hub
+
+> A command-line wrapper for git that makes you better at GitHub.
+> The commands can also be used using "git" instead of "hub".
+
+- Clone a repository you own:
+
+`hub clone {{ repo_name }}`
```
Comment (igorshubovych) [inline pages/common/hub.md]: Yet one thing. We usually do not add spaces between brackets.

**Comment by brpaz** [comment]
ok. removed the spaces.

**Comment by ostera** [comment]
Looking good! Thanks for the contribution @brpaz.

**Pull request merged**